hnn_core/gui/gui.py

```diff
@@ -1594,5 +1991,9 @@ def launch():
     You can pass voila commandline parameters as usual.
     """
     from voila.app import main
+    import debugpy
```

Comment:
Is this specific for your setup to have the VSCode debugger activate inside the GUI?

Reply:
Yeah, I forgot to remove it